run_create_ramiel_codedeploy.py

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env python3
 
 from env import env
-from run_common import AWSCli
+from run_common import AWSCli, re_sub_lines
```

Comment:
한 줄로 분리하는건 어떨까요? 디버깅의 이점을 위해 회사 컨벤션에선 한줄 분리를 권고하고 있는 것으로 알고있습니다 🙏 
```suggestion
from run_common import AWSCli
from run_common import re_sub_lines
```